variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/AbstractVariant.java

```diff
@@ -77,7 +77,7 @@
      */
     private final Map<String, Set<String>> hgvs;
 
-    public AbstractVariant(String chromosome, int start, int end, String reference, String alternate) {
+    public AbstractVariant(String chromosome, long start, long end, String reference, String alternate) {
         if (start > end && !(reference.equals("-"))) {
             throw new IllegalArgumentException("End position must be greater than the start position");
```

Comment:
This message is pretty confusing: it checks `start > end`, then it complains that end must be greater than start.